examples/file_scan/file_scan_request_upload_url_put/src/main/java/cloud/pangeacyber/examples/App.java

```diff
@@ -61,7 +58,7 @@ public static void main(String[] args) throws Exception{
 
 		AcceptedResponse acceptedResponse = client.requestUploadURL(request);
 
-        // extract upload url and upload details that should be posted with the file
+        // extract upload url that should be posted with the file
```

Comment:
URL shouldn't post WITH the file, we should put/post the file to the URL. Isn't it?